source/client/CMakeLists.txt

```diff
@@ -213,24 +213,57 @@ endif()
 add_executable(${QFUSION_CLIENT_NAME} ${CLIENT_BINARY_TYPE} ${CLIENT_HEADERS} ${CLIENT_PLATFORM_HEADERS} ${CLIENT_COMMON_SOURCES} ${CLIENT_PLATFORM_SOURCES} ${BUNDLE_RESOURCES})
 add_dependencies(${QFUSION_CLIENT_NAME} angelwrap cgame cin ftlib game ref_gl snd_openal steamlib ui)
 
+execute_process(COMMAND git rev-parse HEAD WORKING_DIRECTORY ${CMAKE_HOME_DIRECTORY}/.. OUTPUT_VARIABLE GIT_COMMIT_HASH OUTPUT_STRIP_TRAILING_WHITESPACE)
+file(WRITE ${CMAKE_HOME_DIRECTORY}/commit.txt ${GIT_COMMIT_HASH})
```

Comment:
Usually stuff like this is written in the output directory, so that it does not get committed accidentally.

Reply:
steam_appid.txt and commit.txt should probably be added to .gitignore if they haven't been already. Lastly, if steam_appid.txt and commit.txt do not exist and you don't BUILD_STEAM, will it error out? I believe this happened to me, but I'll have to report back. Thanks.

Reply:
I stand corrected. The only issue is cmake generates steam_appid.txt even if BUILD_STEAM isn't used.